KO: Translations for ko/beta/tutorials/estimators/linear.ipynb

cre8tor opened the pull request:
Translation for tutorials/estimators/linear.ipynb

EN ver.
https://github.com/tensorflow/docs/tree/master/site/en/r2/tutorials/estimators/linear.ipynb

Thank you!

cre8tor commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "VqDKQLZn8L-B"
+   },
+   "source": [
+    "## 모델을 위한 특징 공학(feature engineering)\n",
```
I should correct all 특징 -> 특성
Thank you!

cre8tor commented:
> Great translation! :)
Thanks for the review 😄

lamberta commented:
I think we're good here?
Can someone add an 'LGTM' comment or mark approved using the GitHub UI? Thank you

rickiepark commented:
@lamberta I’ll review it in the weekend and let you know approval. :)

cre8tor commented:
@lamberta 'Download notebook' button has added recently, so I also added it.
It seems that the link of 'Download notebook' (https://storage.googleapis.com/tensorflow_docs/site/en...)  supports only English version.
I fixed the path for Korean version(site/ko/...) of existing notebook files, but they didn't work.
I also fixed the link of 'Download notebook' of this file, hoping that it may support sometime soon.
Is it OK to leave broken link for Korean version notebook? If it's not OK, I'll delete 'Download notebook',
 otherwise, I'll add the link to all notebook files.
Thank you.

Choiuijin1125 commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,682 @@
+   },
+   "outputs": [],
+   "source": [
+    "!pip install tensorflow==2.0.0-beta0\n",
```
Now we can available beta1 :) 
So 'beta0' → 'beta1' thanks 👍

cre8tor replied:
Thanks! I fixed version number

lamberta commented:
Thanks.
@yashk2810 recently added the "Download notebook" button and can give an idea when these will be available for translations.

yashk2810 commented:
The download button is available only for TF2 notebooks. So if you add the links for the buttons for TF2 notebooks, @lamberta can enable it.

lamberta commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,682 @@
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/ko/beta/tutorials/estimators/linear.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃허브(GitHub) 소스 보기</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://storage.googleapis.com/tensorflow_docs/site/ko/beta/tutorials/estimators/linear.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />노트북 다운로드</a>\n",
```
Remove button for now.
We can add them to the beta notebooks all at once when it's enabled for translations.

cre8tor replied:
Thank you @lamberta 
Download notebook button has been removed

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "8JSa_duD4tFZ"
+   },
+   "source": [
+    "훈련셋은 627개의 사례로 평가셋은 264개의 사례로 구성되어 있습니다."
```
Sorry for late review. :)
example means sample.
I suggest '사례' --> '샘플'

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "FXJhGGL85TLp"
+   },
+   "source": [
+    "여자는 남자보다 살아남을 확률이 훨씬 높습니다. 이는 모델에서 명확하게 예측가능한 특성입니다."
```
Predictive feature in this context means valuable feature for prediction.
I suggest '모델에서 명확하게 예측가능한' --> '명확하게 모델 예측에 유용한' or '명확하게 모델에 유용한'

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "## 모델을 위한 특성 공학(feature engineering)\n",
+    "추정기는 [특성 열(feature columns)](https://www.tensorflow.org/guide/feature_columns)이라는 시스템을 사용하여 모델이 각각의 입력 특성을 어떻게 해석할지 설명합니다. 추정기가 숫자 입력 벡터를 요구하면, *특성 열*은 모델이 어떻게 각 특성을 변환해야하는지 설명합니다.\n",
+    "\n",
+    "효과적인 모델 학습에서는 적절한 특성 열을 고르고 다듬는 것이 키포인트 입니다. 하나의 특성 열이 원래의 특성 `dict` (*기준 특성 열*)의 원래 입력 중 하나가 되거나 하나 또는 여러 기준열 (*얻어진 특성 열*) 에 정의된 변환을 이용하여 생성된 새로운 열이 될 수 있습니다.\n",
```
"A feature column can be either one of the raw inputs in the original features dict" can be misunderstood easily.
You can read "A feature column can be **made by** the raw inputs in the original features dict".
I just want to remove duplicate '원본' and write '딕셔너리' with dict.
I make it short because this sentense is quite a long.
I suggest '하나의 특성 열은 특성 딕셔너리(dict)의 원본 입력으로 만들어진 열(기본 특성 열)이거나 하나 이상의 기본 열(얻어진 특성 열)에 정의된 변환을 이용하여 새로 생성된 열입니다.' :)

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "\n",
+    "효과적인 모델 학습에서는 적절한 특성 열을 고르고 다듬는 것이 키포인트 입니다. 하나의 특성 열이 원래의 특성 `dict` (*기준 특성 열*)의 원래 입력 중 하나가 되거나 하나 또는 여러 기준열 (*얻어진 특성 열*) 에 정의된 변환을 이용하여 생성된 새로운 열이 될 수 있습니다.\n",
+    "\n",
+    "선형 추정기는 수치형, 범주형 특성을 모두 사용할 수 있습니다. 특성 열은 모든 텐서플로 추정기와 함께 작동하고 목적은 모델링에 사용되는 특성들을 정의하는 것입니다. 또한 원핫인코딩(one-hot-encoding), 정규화(normalization), 버킷화(bucketization)와 같은 특성 공학 방법을 지원합니다."
```
How about '원-핫-인코딩' or '원-핫 인코딩'? :)

cre8tor replied:
Thanks. I fixed it to '원-핫-인코딩' to keep original expression :)

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "puZFOhTDkblt"
+   },
+   "source": [
+    "### 기준 특성 열"
```
Base is difficault to translate. Maybe base means basic in this context.
I suggest '기준' --> '기본'.

cre8tor replied:
It was not easy to translate 'base'
Thanks for suggestion!

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "f4zrAdCIjr3s"
+   },
+   "source": [
+    "`DenseFeatures`는 조밀한(dense) 텐서만 허용합니다. 우선 범주 열을 지시 열로 변환해야 합니다:"
```
indicator_column makes multi-hot representation.
If indicator column is translated as it is, it's hard to understand.
'to inspect a categorical column' is omitted.
I suggest '우선 범주 열을 지시 열로 변환해야' --> '범주형 데이터를 점검하려면 우선 범주형 열에 indicator_column 함수를 적용해야'

cre8tor replied:
👍  This is the translation of 'seeing the forest'. 
Thank you for the guidance.

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "MEp59g5UkHYY"
+   },
+   "source": [
+    "모든 기본 특성을 모델에 추가한 다음에 모델을 훈련해 봅시다. 모델을 훈련하려면 `tf.estimator` API를 이용한 커맨드 하나면 충분합니다:"
```
How about '커맨드 하나면' --> '메서드 호출 한 번이면'? :)

rickiepark commented on site/ko/beta/tutorials/estimators/linear.ipynb:
```diff
@@ -0,0 +1,679 @@
+    "id": "8_eyb9d-ncjH"
+   },
+   "source": [
+    "이제 훈련 모델을 이용해서 평가셋에서 승객에 대해 예측을 할 수 있습니다. 텐서플로 모델은 한번에 사례의 한 배치 또는 콜렉션에 대한 예측을 하도록 최적화되어있습니다. 이전에, `eval_input_fn`은 모든 평가셋을 사용하도록 정의되어 있었습니다."
```
'사례' --> '샘플'
For better reading,
I suggest '사례의 한 배치 또는 콜렉션에' --> '샘플의 배치 또는 일부에'
and '이전에' --> '앞서' or '위에서'.

cre8tor commented:
@Choiuijin1125 @rickiepark Thank you for reviewing my translation!
